ci-operator/step-registry/openshift/e2e/test/network-qe/openshift-e2e-test-network-qe-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: openshift-e2e-test-network-qe
+  env:
+  - name: TEST_FILTERS
+    default: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;FDP&
```

Comment:
@anuragthehatter 
`~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;FDP&` should be `~ChkUpgrade&;~NonPreRelease&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&;FDP&` because https://github.com/openshift/openshift-tests-private/pull/16367 select Disruptive case.

if you do not want to run Disruptive case, you could keep current filters.

Reply:
Agree. We need Disruptive as well. thanks for catching it @kuiwang02

Reply:
Also changed `FDP` to `FDPOVNOVN` tag as suggested by @asood-rh . If we long long tag name, it seems good to me

Reply:
Changed tag to `FdpOvnOvs` as discussed in https://github.com/openshift/openshift-tests-private/pull/16367